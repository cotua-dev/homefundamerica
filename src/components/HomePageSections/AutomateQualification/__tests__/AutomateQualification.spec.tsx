import React from 'react';
import { render, RenderResult } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import { default as AutomateQualification } from '..';

describe('AutomateQualification component', (): void => {
    test('renders', (): void => {
        const { container }: RenderResult = render(<AutomateQualification/>);

        expect(container.firstChild).toMatchSnapshot();
    });
});
