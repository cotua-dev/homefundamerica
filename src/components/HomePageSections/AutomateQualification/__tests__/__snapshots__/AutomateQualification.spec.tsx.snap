// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AutomateQualification component renders 1`] = `
<section>
  <div>
    <h2>
      We Made It Fast & Easy
    </h2>
    <form
      action="#"
    >
      <label
        for="learnMore"
      >
        Request More Info
      </label>
      <select
        name="learnMore"
      >
        <option
          value=""
        >
          Select
        </option>
        <option
          value="fha"
        >
          FHA
        </option>
        <option
          value="conventional"
        >
          Conventional
        </option>
        <option
          value="va"
        >
          VA
        </option>
        <option
          value="non-conventional"
        >
          Non-Conventional
        </option>
        <option
          value="jumbo"
        >
          Jumbo/Super Jumbo
        </option>
        <option
          value="portfolio"
        >
          Portfolio
        </option>
        <option
          value="interest-only"
        >
          Interest-Only
        </option>
        <option
          value="commercial"
        >
          Commercial
        </option>
        <option
          value="commercial-cannabis"
        >
          Commercial Cannabis
        </option>
      </select>
      <label
        for="firstName"
      >
        First Name
      </label>
      <input
        name="firstName"
        placeholder="First Name"
        type="text"
        value=""
      />
      <label
        for="lastName"
      >
        Last Name
      </label>
      <input
        name="lastName"
        placeholder="Last Name"
        type="text"
        value=""
      />
      <label
        for="email"
      >
        Email
      </label>
      <input
        name="email"
        placeholder="Enter your e-mail here"
        type="email"
        value=""
      />
      <label
        for="phoneNumber"
      >
        Phone
      </label>
      <input
        name="phoneNumber"
        placeholder="Phone"
        type="tel"
        value=""
      />
      <button
        type="submit"
      >
        Submit
      </button>
    </form>
  </div>
</section>
`;
